connector/mqtt/vernemq/broker/README.md

```diff
@@ -1,55 +1,58 @@
 # **Dojot VerneMQ**
 
-The Dojot VerneMQ service is a extension of [VerneMQ](https://github.com/vernemq/vernemq) with some features for dojot case.
+The Dojot VerneMQ service is an extension of [VerneMQ](https://github.com/vernemq/vernemq) with some
+modifications for Dojot's integration.
 
+# **Configuration**
 
-## **Environment variables**
+There are two configurations for VerneMQ: the one provided by the VerneMQ itself and the Dojot's
```

Comment:
Shoudn't be better 'There are two configuration for the Dojot VerneMQ service: "?